.travis.yml

```diff
@@ -7,8 +7,8 @@ matrix:
   fast_finish: true
```

Comment:
Could you revert the `travis.yml` changes please, we have a organsation wide upgrade solution for this in place :)